docker/shadow-hand/indigo/base/Dockerfile

```diff
@@ -4,15 +4,17 @@ LABEL Description="This ROS Indigo image contains Shadow's dexterous hand softwa
 
 ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$toolset_branch/bin/setup_dev_machine"
 
-RUN echo "Fixing file system" && \
-    mkdir -p /etc/apt/sources.list.d && \
+RUN echo "Fixing file system" && mkdir -p /etc/apt/sources.list.d && \
```

Comment:
you do not need to create folder since you are deleting it. Unless `rm` is returning error code for non existing folder. But it is not it's default behaviour. It simply return 0.

Reply:
Ok, that was just as it was before I started...

Reply:
github is still seen changes there :)